addon/webextension/background/main.js

```diff
@@ -65,13 +63,13 @@ window.main = (function () {
   });
 
   communication.register("openMyShots", () => {
-    chrome.tabs.create({url: backend + "/shots"});
+    browser.tabs.create({url: backend + "/shots"});
   });
 
   communication.register("openShot", ({url, copied}) => {
     if (copied) {
       const id = makeUuid();
-      chrome.notifications.create(id, {
+      browser.notifications.create(id, {
```

Comment:
This also returns a promise, please wrap in a `watchPromise` call.

The [MDN Page](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/notifications/create) for `browser.notifications.create` contains a rather grim warning: "If you call notifications.create() more than once in rapid succession, Firefox may end up not displaying any notification at all."

I don't know how quickly openShot might return, but would it be a good idea to debounce this function?